backend/src/main/java/org/cryptomator/hub/license/LicenseHolder.java

```diff
@@ -75,6 +85,45 @@ public void set(String token) throws JWTVerificationException {
 		settings.persist();
 	}
 
+	/**
+	 * Attempts to refresh the Hub licence every day at 01:00 UTC if claim refreshURL is present.
+	 */
+	@Scheduled(cron = "0 1 * * * ?", timeZone = "UTC")
+	void refreshLicenseScheduler() throws InterruptedException {
+		if (license != null) {
+			var refreshUrl = licenseValidator.refreshUrl(license.getToken());
+			if (refreshUrl.isPresent()) {
+				var client = HttpClient.newBuilder().followRedirects(HttpClient.Redirect.NORMAL).build();
+				refreshLicense(refreshUrl.get(), license.getToken(), client);
+			}
+		}
+	}
+
+	//visible for testing
+	void refreshLicense(String refreshUrl, String license, HttpClient client) throws InterruptedException {
+		var parameters = new HashMap<String, String>();
+		parameters.put("token", license);
```

Comment:
Use `Map.of(...)`

Reply:
Fixed in [174c7aa](https://github.com/cryptomator/hub/pull/246/commits/174c7aad99b66f5373fc5c46b696b96ec41ad1d2)